cmake/protobuf-config.cmake.in

```diff
@@ -1,11 +1,17 @@
 # Version info variables
 set(PROTOBUF_VERSION        "@protobuf_VERSION@")
 set(PROTOBUF_VERSION_STRING "@protobuf_VERSION_STRING@")
+set(PROTOBUF_USE_ZLIB       "@protobuf_WITH_ZLIB@")
+
 
 # Current dir
 get_filename_component(_PROTOBUF_PACKAGE_PREFIX
   "${CMAKE_CURRENT_LIST_FILE}" PATH)
 
+if (PROTOBUF_USE_ZLIB)
+  find_package(ZLIB)
```

Comment:
I think it make sense to add `REQUIRED` here since if user don't want to use `ZLIB` he can add `PROTOBUF_USE_ZLIB=OFF`